nickel-mode.el

```diff
@@ -0,0 +1,103 @@
+;;; nickel-mode.el --- A major mode for editing Nickel source code -*-lexical-binding: t-*-
+
+;; Copyright (c) Tweag Holding and its affiliates.
```

Comment:
I don't know if there's a Tweag Holding corporate entity these days — probably better to just write Tweag here, or Tweag EURL (which I know _is_ a real company, because they pay me).

Reply:
Right. I got that from the main Nickel repo's [license](https://github.com/tweag/nickel/blob/master/LICENSE). Maybe that is wrong too?

Reply:
Could be. But I suppose @aspiwack could comment most authoritatively.

Reply:
The correct copyright holder, per policy, should be _Tweag I/O limited_. (see _e.g._ the [project template](https://github.com/tweag/project/blob/master/LICENSE)).